package.json

```diff
@@ -32,5 +41,10 @@
   "bugs": {
     "url": "https://github.com/danfinlay/eth-method-registry/issues"
   },
-  "homepage": "https://github.com/danfinlay/eth-method-registry#readme"
+  "homepage": "https://github.com/danfinlay/eth-method-registry#readme",
+  "files": [
+    "tape",
+    "registry-map.json",
+    "abi.json"
+  ]
```

Comment:
We should delete the `files` property for now.

`files` is used by `npm` to identify which files (other than some defaults) should be included in the package bundle when you run `npm publish` or `npm pack` (try this with `npm pack --dry-run`). We should add `files` with the appropriate values when we convert to TypeScript, but we don't need it for this PR.

At that time, the contents of the `dist` folder and any imported `.json` files should be included. (I'm not sure if the imported JSON files will be copied to `dist`.)